services/hooks/config.yml

```diff
@@ -2,9 +2,9 @@ defaults:
   # defaults outlining all configuration options and some sane defaults
   app:
     component:        hooks
-    hookTableName:     !env HOOK_TABLE_NAME
-    lastFireTableName: !env LASTFIRE_TABLE_NAME
-    queuesTableName:  !env QUEUE_TABLE_NAME
+    hookTableName: Hooks
+    lastFireTableName: LastFire3
```

Comment:
Will this require a corresponding change to the static client?  And, do I recall that changes to the static clients don't get deployed automatically?

Reply:
Why will changing the table name require updating clients? I think I'm missing something or other here.

Reply:
The hooks client has scope `auth:azure-table:read-write:${azureAccountId}/LastFire`, not `auth:azure-table:read-write:${azureAccountId}/LastFire3`

Reply:
Oh, those clients! haha, I was thinking of every other client for some reason. Yes, good catch.